src/main/java/nl/hsac/fitnesse/fixture/util/HttpClient.java

```diff
@@ -49,6 +51,22 @@ public void post(String url, HttpResponse response, Map<String, Object> headers,
         methodPost.setEntity(ent);
         getResponse(url, response, methodPost, headers);
     }
+
+    public void post(String url, HttpResponse response, Map<String, Object> headers, File file) {
+        HttpPost methodPost = new HttpPost(url);
+
+
+        MultipartEntityBuilder builder = MultipartEntityBuilder.create();
+        builder.addBinaryBody(file.getName(), new File(file.getAbsolutePath()),
```

Comment:
Why do we need `new File(file.getAbsolutePath())`, instead of just using `file`?